src/components/functional/toasts-list/toast-list.tsx

```diff
@@ -12,19 +12,30 @@ export class ToastList {
   private eventBus: IEventBus = new EventBus();
   @Prop() transactionToasts: ITransactionToast[];
   @Prop() customToasts: CustomToastType[];
+  @Prop() maxTransactions: number = 5;
 
   @Method() async getEventBus() {
     return this.eventBus;
   }
 
+  private handleCustomToastDelete(toastId: string) {
+    this.eventBus.publish(ToastEventsEnum.CLOSE_TOAST, toastId);
+  }
+
+  private handleTransactionToastDelete(toastId: string) {
+    this.eventBus.publish(ToastEventsEnum.CLOSE_TOAST, toastId);
+  }
+
   render() {
     return (
       <div class="toast-list" id="toast-list">
-        {this.customToasts?.map(toast => (
-          <generic-toast toast={toast} onHandleDeleteToast={() => this.eventBus.publish(ToastEventsEnum.CLOSE_TOAST, toast.toastId)}></generic-toast>
-        ))}
+        {this.customToasts?.map(toast => <generic-toast toast={toast} onHandleDeleteToast={this.handleCustomToastDelete.bind(this, toast.toastId)}></generic-toast>)}
         {this.transactionToasts?.map(toast => (
-          <transaction-toast {...toast} onHandleDeleteToast={() => this.eventBus.publish(ToastEventsEnum.CLOSE_TOAST, toast.toastId)}></transaction-toast>
+          <transaction-toast
```

Comment:
maxTransactions why pass from outside ?

Reply:
Because we must let it configurable from controller or a config maybe